Sources/MapboxDirectionsCLI/CodingOperation.swift

```diff
@@ -40,6 +73,24 @@ class CodingOperation<ResponceType : Codable, OptionsType : DirectionsOptions >
         }
     }
     
+    #if !os(Linux)
+    private func interpolate(route: Route) -> [CLLocationCoordinate2D?] {
+        guard route.expectedTravelTime > 0, let polyline = route.shape,
+              let firstCoordinate = polyline.coordinates.first else { return [] }
+        
+        var distanceAway: CLLocationDistance = 0
+        let distance = route.distance/route.expectedTravelTime
+        var interpolatedCoordinates = [firstCoordinate]
+        while distanceAway <= route.distance {
+            if let nextCoordinate = polyline.coordinateFromStart(distance: distanceAway) {
+                interpolatedCoordinates.append(nextCoordinate)
+            }
+            distanceAway += distance
+        }
+        return interpolatedCoordinates
```

Comment:
Not sure if it's important but we can get a bit off the final coordinate. What about adding the `polyline.coordinates.last` into `interpolatedCoordinates`?